pkg/standalone/runconfig/run_config.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runconfig
+
+import "github.com/dapr/cli/pkg/standalone"
+
+type AppsRunConfig struct {
+	Common  Common `yaml:"common"`
+	Apps    []Apps `yaml:"apps"`
+	Version int    `yaml:"version"`
+}
+
+type Apps struct {
+	standalone.RunConfig `yaml:",inline"`
+	AppDir               string     `yaml:"app_dir"`
```

Comment:
```suggestion
	AppDirPath               string     `yaml:"app_dir_path"`
```
see https://github.com/dapr/cli/issues/1141#issuecomment-1360046093